src/tools/pontoon/deactivating_users.md

```diff
@@ -0,0 +1,10 @@
+# Deactivating Pontoon users
+
+Access Django’s admin interface at `https://pontoon.mozilla.org/a/` (note that this is not the usual admin interface), then click `Users`. In the next page search for the user you want to deactivate (safer to search by email address):
+
+* Click on the user among the results displayed under the search field.
+* In the *Permissions* section, deselect the `Active` checkbox and click `SAVE` at the bottom of the page.
+
+This will prevent them from logging in in Pontoon, and disconnect them immediately from the system.
```

Comment:
I think it's grammatically (#1) and semantically (#2) more correct like this:
1. ...from logging **into** Pontoon...
1. ...and **log them out** immediately...

But I'm not sure, so feel free to ignore if you disagree.

Reply:
"This will prevent them from _logging into Pontoon_, and _will log them out of the system immediately_."

Reply:
Thanks, fixing it :-)

I kind of remember Jeff writing "logging in in something", so I called for help.

Reply:
I are well English sporker ;-)